server/core/src/https/views/apps.rs

```diff
@@ -27,20 +25,53 @@ struct AppsView {
 #[derive(Template)]
 #[template(path = "apps_partial.html")]
 struct AppsPartialView {
-    // todo - actually list the applications the user can access here.
+    apps: Vec<AppLink>,
+}
+
+fn get_transformer(op_id: Uuid) -> impl Fn(OperationError) -> Response {
+    move |error| {
+        match error {
+            OperationError::SessionExpired => {
+                HtmlTemplate(LoginView {
+                    username: "",
+                    remember_me: false,
+                }).into_response()
+            }
+            _ => {
+                HtmlTemplate(UnrecoverableErrorView {
+                    err_code: error,
+                    operation_id: op_id,
+                }).into_response()
+            }
+        }
+    }
 }
 
 pub(crate) async fn view_apps_get(
-    State(_state): State<ServerState>,
-    Extension(_kopid): Extension<KOpId>,
+    State(state): State<ServerState>,
+    Extension(kopid): Extension<KOpId>,
     HxRequest(hx_request): HxRequest,
-    VerifiedClientInformation(_client_auth_info): VerifiedClientInformation,
+    VerifiedClientInformation(client_auth_info): VerifiedClientInformation,
```

Comment:
https://docs.rs/axum/latest/axum/response/type.Result.html

We should use this here I think to handle the possible app_links error.